fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/physical/PhysicalAssertOneRowOperator.java

```diff
@@ -55,14 +55,7 @@ public <R, C> R accept(OptExpressionVisitor<R, C> visitor, OptExpression optExpr
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        PhysicalAssertOneRowOperator that = (PhysicalAssertOneRowOperator) o;
-        return checkRows == that.checkRows && assertion == that.assertion && Objects.equal(tips, that.tips);
+        return this == o;
```

Comment:
if you change the equals method, the hashCode method should also be changed

Reply:
done